trackintel/model/util.py

```diff
@@ -189,6 +137,7 @@ class TrackintelGeoDataFrameWithFallback(GeoDataFrame):
     Fallback to DataFrame if looses geometry and _check fails.
```

Comment:
TO clarify: as we set fallback_class = None, the 1st and 3rd case at the moment is the same and correspond to the checks in line 154?

Reply:
Your understanding is correct. I adapted the docstring to explain the change a bit better. Can you tell me if it is clearer now? :)